cfgov/unprocessed/css/molecules/global-search.less

```diff
@@ -267,6 +258,22 @@
     } );
 }
 
+.m-global-search {
+    // Tab trigger is used to capture press of the tab key so that
+    // global search can be collapsed when it hits this element.
```

Comment:
That is correct. I guess it's a little weird in that after the menu closes the focus jumps back up to the beginning of the page. Not sure if that's an issue as it's still all tab-accessible (e.g. after collapsing, continue tabbing through the page past the search button). Perhaps focus should jump back to the closed search button? I'll push a fix for this.

This approach is to avoid a lot of complicated logic on tabbing behavior. The search box needs to close after tabbing from the search button at desktop and mobile size, but needs to instead close after the last of the suggested topics links at tablet size. I believe once the bewd-side comes in, the suggested topics links will not be static. Therefore there'd need to be logic around adding and removing events at different breakpoints and on the bewd-side to identify the last link, if present.